@xen-orchestra/web-core/lib/components/UiTag.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <span class="ui-tag typo p3-regular" :class="[color, { light }]">
+    <slot name="icon">
+      <UiIcon :icon fixed-width />
+    </slot>
+    <slot />
```

Comment:
To keep consistency with UiChip, I think we should wrap the content in a tag, with `overflow` + `text-ellipsis`

```suggestion
    <span class="content"><slot /></span>
```